kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/schemaregistry/SchemaRegistryAwareRecordSerDe.java

```diff
@@ -63,14 +65,29 @@
 
   private static SchemaRegistryClient createSchemaRegistryClient(KafkaCluster cluster) {
     Objects.requireNonNull(cluster.getSchemaRegistry());
+    Objects.requireNonNull(cluster.getSchemaRegistry().getUrl());
     List<SchemaProvider> schemaProviders =
         List.of(new AvroSchemaProvider(), new ProtobufSchemaProvider(), new JsonSchemaProvider());
-    //TODO add auth
+
+    Map<String, String> configs = new HashMap<>();
+    String username = cluster.getSchemaRegistry().getUsername();
+    String password = cluster.getSchemaRegistry().getPassword();
+
+    if (username != null && password != null) {
+      configs.put("basic.auth.credentials.source", "USER_INFO");
+      configs.put("basic.auth.user.info", username + ":" + password);
```

Comment:
Could you please move these properties to constants or replace them with existing ones?

Reply:
I meant `basic.auth.user.info` and basic.auth.credentials.source

Reply:
Done.
I used constants stored in `io.confluent.kafka.serializers.AbstractKafkaSchemaSerDeConfig`.